src/cli.js

```diff
@@ -10,14 +10,14 @@ const exec = require('child_process').execSync
 const dedent = require('dedent-js')
 const getSymlinkedDependencies = require('./getSymlinkedDependencies')
 const getMetroConfig = require('./getMetroConfig')
-const getDependencyPath = require('./getDependencyPath')
 
 const CONFIG_FILENAME = 'rn-cli.config.js'
 
-const mapDep = dep => `    - ${dep} -> ${getDependencyPath(dep)}`
+const mapDep = dep => `    - ${dep}`
```

Comment:
🚧 Temp change to handle the fact that we're working in paths now, not package names. Needs cleanup.